appverifier/src/main/java/com/android/mdl/appreader/MainActivity.kt

```diff
@@ -28,6 +34,11 @@ class MainActivity : AppCompatActivity() {
     private val navController by lazy {
         Navigation.findNavController(this, R.id.nav_host_fragment)
     }
+    private lateinit var fusedLocationClient: FusedLocationProviderClient
+
+    object Const {
+        const val LOCATION_REQUEST_CODE = 123
```

Comment:
much like Holder app, I'm not focusing on the back and forth of the Location permission denying then granting -- user is asked once at the start of the app, if they decline Location permission then the recorded lat/long will be (0,0)